Source/Core/Core/HW/WiimoteEmu/EmuSubroutines.cpp

```diff
@@ -566,6 +567,7 @@ void Wiimote::DoState(PointerWrap& p)
   // Sub-devices:
   m_speaker_logic.DoState(p);
   m_camera_logic.DoState(p);
+  m_camera_logic.SetEnabled(m_status.ir);
```

Comment:
Hm, should this check whether we're reading or writing (and only call this when reading)?
Then again, I suppose it should have no consequence even if we do it during writing as well (since it _should_ have the same value anyways).

Reply:
I've made the suggested change.